frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -119,130 +107,83 @@ export default function SideNavigation({
+            />
+            <NavMenuItem
+              href='/beta/review'
+              selectedPage={page}
+              primaryText='Reviews'
+              drawerOpen={drawerOpen}
+              menuPage='beta/review'
+              title='Review'
+              icon={<ListAltIcon />}
+              badgeCount={reviewCount}
+            />
+            <Divider />
+            <NavMenuItem
+              href='/beta/docs/api'
+              selectedPage={page}
+              primaryText='API'
+              drawerOpen={drawerOpen}
+              menuPage='beta/api'
+              title='API'
+              icon={<LinkIcon />}
+            />
+            <NavMenuItem
+              href='/beta/help'
+              selectedPage={page}
+              primaryText='Support'
+              drawerOpen={drawerOpen}
+              menuPage='beta/help'
+              title='Help & Support'
+              icon={<ContactSupportIcon />}
+            />
+            <Divider />
+            {/* TODO Once currentUser api has been updated to use roles we should check if they're admin */}
+            {currentUser && (
+              <>
+                <NavMenuItem
+                  href='/beta/schema/list'
+                  selectedPage={page}
+                  primaryText='Schemas'
+                  drawerOpen={drawerOpen}
+                  menuPage='beta/schemas'
+                  title='Schemas'
+                  icon={<SchemaIcon />}
+                />
+                <Divider />
+              </>
+            )}
           </StyledList>
-          <Divider />
           <StyledList>
-            <ListItem disablePadding>
-              <Link href='/beta/docs/api'>
-                <ListItemButton selected={page === 'api'} data-test='apiDocsLink'>
-                  <ListItemIcon>
-                    {!drawerOpen ? (
-                      <Tooltip title='API' arrow placement='right'>
-                        <LinkIcon />
-                      </Tooltip>
-                    ) : (
-                      <LinkIcon />
-                    )}
-                  </ListItemIcon>
-                  <ListItemText primary='API' />
-                </ListItemButton>
-              </Link>
-            </ListItem>
-            <ListItem disablePadding>
-              <Link href='/beta/help'>
-                <ListItemButton selected={page === 'help'} data-test='supportLink'>
-                  <ListItemIcon>
-                    {!drawerOpen ? (
-                      <Tooltip title='Help & Support' arrow placement='right'>
-                        <ContactSupportIcon />
-                      </Tooltip>
-                    ) : (
-                      <ContactSupportIcon />
-                    )}
-                  </ListItemIcon>
-                  <ListItemText primary='Support' />
-                </ListItemButton>
-              </Link>
-            </ListItem>
+            <Divider />
+            <MenuItem>
+              <IconButton
```

Comment:
Could we add a link to the user settings here.  I _realise_ it's in the top right, and I _realise_ people should see it's there.  However a duplicate link at the bottom here doesn't seem prohibitively complex and I think it'd provide a much more obvious (permanently visible) link.